gems/aws-sdk-lambda/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Add RBS signature files to support static type checking.
+
```

Comment:
Our release system will detect changes and bump all of the gems, but I think adding these for all gems will be helpful (we can do that step at the end).

Reply:
When we build the gems, if we detect any changes through `git diff`, it becomes 
```
* Feature - Code Generated Changes, see `./build_tools` or `aws-sdk-core`'s CHANGELOG.md for details.
```
and is handled automatically, but this is not descriptive, we probably will want to add manual entries like you did for lambda to all services, but we can do that much later.

Reply:
Depending on the release timing for this, we may want to just improve the automated release system message in those cases (ie: reference the change log from build tools).